src/back/backend_handler.rs

```diff
@@ -207,6 +207,14 @@ impl BackEndHandler {
                 trace!("list_authenticators egress");
                 self.result_to_response(NativeResult::ListAuthenticators(result), header)
             }
+            NativeOperation::ListKeys(op_list_keys) => {
+                let app_name =
```

Comment:
The check I was talking in the other comment could be here but could also be in the `is_capable` method where we would check if the request's opcode is a Core or a Crypto operation and based on that check if the provider can do it or not.